src/plugins/kibana/public/discover/index.html

```diff
@@ -1,5 +1,53 @@
 <div ng-controller="discover" class="app-container">
-  <navbar name="discover">
+  <navbar name="discover-options">
+    <div class="discover-info">
+      <span ng-show="opts.savedSearch.id" class="discover-info-title">
+        <span ng-bind="::opts.savedSearch.title"></span>
+        <i aria-label="Reload Saved Search" tooltip="Reload Saved Search" ng-click="resetQuery();" class="fa fa-undo small"></i>
+      </span>
+
+      <strong class="discover-info-hits">{{(hits || 0) | number:0}}</strong>
+      <ng-pluralize count="hits" when="{'1':'hit', 'other':'hits'}"></ng-pluralize>
+    </div>
+    <div class="discover-actions button-group" role="toolbar">
+      <button
+        ng-click="newQuery()"
+        aria-label="New Search">
+        <span>New</span>
+      </button>
+      <button
+        ng-click="configTemplate.toggle('save');"
+        ng-class="{active: configTemplate.is('save')}"
+        aria-haspopup="false"
+        aria-expanded="{{ configTemplate.is('save') }}"
+        aria-label="Save Search">
+        <span>Save</span>
+      </button>
+      <button
+        aria-haspopup="false"
+        aria-expanded="{{ configTemplate.is('load') }}"
+        ng-click="configTemplate.toggle('load');"
+        ng-class="{active: configTemplate.is('load')}"
+        aria-label="Load Saved Search">
+        <span>Open</span>
+      </button>
+      <button
+        aria-label="Share Search"
+        aria-haspopup="false"
+        aria-expanded="{{ configTemplate.is('share') }}"
+        ng-class="{active: configTemplate.is('share')}"
+        ng-click="configTemplate.toggle('share');">
+        <span>Share</span>
+      </button>
+    <div class="chrome-actions" kbn-chrome-append-nav-controls></div>
```

Comment:
I think this is currently used for injecting the logout button in the Shield plugin right? But we want that moved to the side bar